From a6b250054e5bc27b87414c860c9b808a4beef552 Mon Sep 17 00:00:00 2001
From: Awais Belal <awais_belal@mentor.com>
Date: Tue, 16 Aug 2016 16:07:45 +0500
Subject: [PATCH] obey CMAKE_INSTALL_LIBDIR

If the path to CMAKE_INSTALL_LIBDIR is not followed appropriately
the installation will not work correctly on a multilib platofrm.

Signed-off-by: Awais Belal <awais_belal@mentor.com>
---
 source/CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/source/CMakeLists.txt b/source/CMakeLists.txt
index de227d6..73672a1 100644
--- a/source/CMakeLists.txt
+++ b/source/CMakeLists.txt
@@ -198,5 +198,5 @@ target_include_directories(${SPIRV_TOOLS}
 
 install(TARGETS ${SPIRV_TOOLS}
   RUNTIME DESTINATION bin
-  LIBRARY DESTINATION lib
-  ARCHIVE DESTINATION lib)
+  LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
+  ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR})
-- 
1.9.1

